label_studio_converter/converter.py

```diff
@@ -659,6 +682,18 @@ def add_image(images, width, height, image_id, image_path):
                             'area': get_polygon_area(x, y),
                         }
                     )
+                elif 'brushlabels' in label and brush.pycocotools_imported:
+                    segmentation = brush.ls_rle_to_coco_rle(label["rle"], height, width)
+                    annotations.append(
+                        {
+                            "image_id": image_id,
```

Comment:
Hi, I am a Label Studio user and thank you so much for your PR.
I tried clone and tried it, but I got an error.
Is the `annotation_id` missing from this line here?

```
{
  "id": annotation_id,
  "image_id": image_id,
  "segmentation": segmentation,
  "area": brush.get_cocomask_area(segmentation),
  "bbox": brush.get_cocomask_bounding_box(segmentation),
  "iscrowd": 1,
  "category_id": category_id,
}
```

Reply:
Good catch. Just added the missing key.